main.go

```diff
@@ -314,6 +316,24 @@ func createIssueComment(c *gin.Context, db *sql.DB) {
 	IssueNumber := c.PostForm("issue_number")
 	fmt.Println(c.PostForm("comment_and_close"))
 
+	currentRepo, err := getCurrentRepo(db, username, repoName)
+	if err != nil {
+		fmt.Println(err)
+		c.AbortWithStatus(http.StatusInternalServerError)
+		return
+	}
+
+	writeAccess, err := hasWriteAccess(db, currentRepo, currentUser)
+	if err != nil {
+		fmt.Println(err)
+		c.AbortWithStatus(http.StatusInternalServerError)
+		return
+	}
+	if !writeAccess {
```

Comment:
Why did we add this here at all ? This is create issue comment any authorized user should be able to comment.

Reply:
Done